spec/requests/api/v1/items_request_spec.rb

```diff
@@ -35,11 +35,11 @@
       name: "Organic Crash Pad",
       description: "Super soft and thicc, heavy though",
       count: 1,
-      category: 7,
+      category: "Crash Pads",
```

Comment:
Same question here as above. I thought the BE database is storing ints not strings? Maybe I don't understand enums...